Doc/faq/general.rst

```diff
@@ -157,7 +155,11 @@ unreleased versions, built directly from the CPython development repository.  In
 practice, after a final minor release is made, the version is incremented to the
 next minor version, which becomes the "a0" version, e.g. "2.4a0".
 
-See also the documentation for :data:`sys.version`, :data:`sys.hexversion`, and
+See the `Developer's Guide
+<https://devguide.python.org/developer-workflow/development-cycle/>__`
```

Comment:
<img width="783" alt="image" src="https://github.com/python/cpython/assets/1324225/d726a9fc-3a61-4e41-8118-df78ac6492f0">

reST links are always a bit tricky, I think this should do it:

```suggestion
See the `Developer's Guide
<https://devguide.python.org/developer-workflow/development-cycle/>`__
```